File: shared/src/main/java/ru/privatenull/pnauth/kernel/event/EventDispatchRunner.java
package ru.privatenull.pnauth.kernel.event;
/** Internal dispatch bridge used by event-bus implementations. */
public final class EventDispatchRunner {
    private EventDispatchRunner() { }
    public static void run(ListenerOptions options, Runnable action) { EventDispatchScope.run(options, action); }
}

File: shared/src/main/kotlin/ru/privatenull/pnauth/kernel/event/EventDispatchRunner.kt
package ru.privatenull.pnauth.kernel.event

/** Internal dispatch bridge used by event-bus implementations. */
object EventDispatchRunner {
    @JvmStatic
    fun run(options: ListenerOptions, action: Runnable) {
        EventDispatchScope.run(options, action)
    }
}
